test source distribution / test_sdist_preview (3.10, arm64) fails

```
2025-03-02T08:54:42.9539070Z Current runner version: '2.322.0'
2025-03-02T08:54:42.9555000Z ##[group]Operating System
2025-03-02T08:54:42.9555480Z macOS
2025-03-02T08:54:42.9555800Z 14.7.2
2025-03-02T08:54:42.9556110Z 23H311
2025-03-02T08:54:42.9556410Z ##[endgroup]
2025-03-02T08:54:42.9556810Z ##[group]Runner Image
2025-03-02T08:54:42.9557170Z Image: macos-14-arm64
2025-03-02T08:54:42.9557490Z Version: 20250120.774
2025-03-02T08:54:42.9558160Z Included Software: https://github.com/actions/runner-images/blob/macos-14-arm64/20250120.774/images/macos/macos-14-arm64-Readme.md
2025-03-02T08:54:42.9559260Z Image Release: https://github.com/actions/runner-images/releases/tag/macos-14-arm64%2F20250120.774
2025-03-02T08:54:42.9560030Z ##[endgroup]
2025-03-02T08:54:42.9560610Z ##[group]Runner Image Provisioner
2025-03-02T08:54:42.9561220Z 2.0.422.1+55c30c14fe2a0a1547db1b656933ae07d97649a9
2025-03-02T08:54:42.9561680Z ##[endgroup]
2025-03-02T08:54:42.9562380Z ##[group]GITHUB_TOKEN Permissions
2025-03-02T08:54:42.9563650Z Contents: read
2025-03-02T08:54:42.9564010Z Metadata: read
2025-03-02T08:54:42.9564360Z ##[endgroup]
2025-03-02T08:54:42.9566350Z Secret source: Actions
2025-03-02T08:54:42.9566860Z Prepare workflow directory
2025-03-02T08:54:42.9968780Z Prepare all required actions
2025-03-02T08:54:43.0012360Z Uses: onnx/onnx/.github/workflows/preview_source_dist_test.yml@refs/heads/main (b3e627c5208a22eace1e3005210c21965025f093)
2025-03-02T08:54:43.0015540Z ##[group] Inputs
2025-03-02T08:54:43.0015930Z   publish_pypi_weekly: no
2025-03-02T08:54:43.0016310Z   publish_testpypi_weekly: yes
2025-03-02T08:54:43.0016700Z   os: macos
2025-03-02T08:54:43.0017020Z ##[endgroup]
2025-03-02T08:54:43.0017490Z Complete job name: test source distribution / test_sdist_preview (3.10, arm64)
2025-03-02T08:54:43.0394610Z ##[group]Run python -m pip uninstall -y onnx-weekly
2025-03-02T08:54:43.0395210Z [36;1mpython -m pip uninstall -y onnx-weekly[0m
2025-03-02T08:54:43.0395650Z [36;1mpython -m pip install setuptools[0m
2025-03-02T08:54:43.0396270Z [36;1mpython -m pip install --use-deprecated=legacy-resolver --no-binary onnx-weekly onnx-weekly[0m
2025-03-02T08:54:43.0396970Z [36;1mpython -m pip install pytest[0m
2025-03-02T08:54:43.0397340Z [36;1mpytest[0m
2025-03-02T08:54:43.1693480Z shell: /bin/bash -e {0}
2025-03-02T08:54:43.1694010Z env:
2025-03-02T08:54:43.1694310Z   MACOSX_DEPLOYMENT_TARGET: 12.0
2025-03-02T08:54:43.1694650Z ##[endgroup]
2025-03-02T08:54:44.2118560Z WARNING: Skipping onnx-weekly as it is not installed.
2025-03-02T08:54:44.6424510Z Collecting setuptools
2025-03-02T08:54:44.6897880Z   Downloading setuptools-75.8.2-py3-none-any.whl.metadata (6.7 kB)
2025-03-02T08:54:44.7080010Z Downloading setuptools-75.8.2-py3-none-any.whl (1.2 MB)
2025-03-02T08:54:44.7575410Z    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 43.2 MB/s eta 0:00:00
2025-03-02T08:54:44.7605170Z Installing collected packages: setuptools
2025-03-02T08:54:45.2648890Z Successfully installed setuptools-75.8.2
2025-03-02T08:54:45.3600570Z 
2025-03-02T08:54:45.3602010Z [notice] A new release of pip is available: 24.3.1 -> 25.0.1
2025-03-02T08:54:45.3602580Z [notice] To update, run: python -m pip install --upgrade pip
2025-03-02T08:54:45.6656510Z Collecting onnx-weekly
2025-03-02T08:54:45.7016110Z   Downloading onnx_weekly-1.18.0.dev20250224.tar.gz (12.2 MB)
2025-03-02T08:54:45.7940460Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.2/12.2 MB 168.1 MB/s eta 0:00:00
2025-03-02T08:54:47.5032010Z   Installing build dependencies: started
2025-03-02T08:54:48.7136980Z   Installing build dependencies: finished with status 'done'
2025-03-02T08:54:48.7145570Z   Getting requirements to build wheel: started
2025-03-02T08:54:48.7467990Z   Getting requirements to build wheel: finished with status 'done'
2025-03-02T08:54:48.7661930Z 
2025-03-02T08:54:48.7662310Z [notice] A new release of pip is available: 24.3.1 -> 25.0.1
2025-03-02T08:54:48.7662730Z [notice] To update, run: python -m pip install --upgrade pip
2025-03-02T08:54:48.7717590Z ERROR: Exception:
2025-03-02T08:54:48.7718010Z Traceback (most recent call last):
2025-03-02T08:54:48.7719390Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/cli/base_command.py", line 105, in _run_wrapper
2025-03-02T08:54:48.7719950Z     status = _inner_run()
2025-03-02T08:54:48.7720490Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/cli/base_command.py", line 96, in _inner_run
2025-03-02T08:54:48.7721040Z     return self.run(options, args)
2025-03-02T08:54:48.7721260Z            ~~~~~~~~^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7721760Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/cli/req_command.py", line 67, in wrapper
2025-03-02T08:54:48.7722280Z     return func(self, options, args)
2025-03-02T08:54:48.7722810Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/commands/install.py", line 379, in run
2025-03-02T08:54:48.7723320Z     requirement_set = resolver.resolve(
2025-03-02T08:54:48.7723690Z         reqs, check_supported_wheels=not options.target_dir
2025-03-02T08:54:48.7723960Z     )
2025-03-02T08:54:48.7724460Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/resolution/legacy/resolver.py", line 181, in resolve
2025-03-02T08:54:48.7725060Z     discovered_reqs.extend(self._resolve_one(requirement_set, req))
2025-03-02T08:54:48.7725360Z                            ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7725930Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/resolution/legacy/resolver.py", line 505, in _resolve_one
2025-03-02T08:54:48.7727540Z     dist = self._get_dist_for(req_to_install)
2025-03-02T08:54:48.7728220Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/resolution/legacy/resolver.py", line 458, in _get_dist_for
2025-03-02T08:54:48.7728850Z     dist = self.preparer.prepare_linked_requirement(req)
2025-03-02T08:54:48.7729490Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/operations/prepare.py", line 527, in prepare_linked_requirement
2025-03-02T08:54:48.7730120Z     return self._prepare_linked_requirement(req, parallel_builds)
2025-03-02T08:54:48.7730450Z            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7731060Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/operations/prepare.py", line 642, in _prepare_linked_requirement
2025-03-02T08:54:48.7731590Z     dist = _get_prepared_distribution(
2025-03-02T08:54:48.7731790Z         req,
2025-03-02T08:54:48.7731990Z     ...<3 lines>...
2025-03-02T08:54:48.7732160Z         self.check_build_deps,
2025-03-02T08:54:48.7732370Z     )
2025-03-02T08:54:48.7734120Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/operations/prepare.py", line 72, in _get_prepared_distribution
2025-03-02T08:54:48.7734750Z     abstract_dist.prepare_distribution_metadata(
2025-03-02T08:54:48.7735020Z     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
2025-03-02T08:54:48.7735270Z         finder, build_isolation, check_build_deps
2025-03-02T08:54:48.7735570Z         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7735800Z     )
2025-03-02T08:54:48.7735930Z     ^
2025-03-02T08:54:48.7736470Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/distributions/sdist.py", line 56, in prepare_distribution_metadata
2025-03-02T08:54:48.7737050Z     self._install_build_reqs(finder)
2025-03-02T08:54:48.7737280Z     ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
2025-03-02T08:54:48.7737830Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/distributions/sdist.py", line 126, in _install_build_reqs
2025-03-02T08:54:48.7738380Z     build_reqs = self._get_build_requires_wheel()
2025-03-02T08:54:48.7739510Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/distributions/sdist.py", line 103, in _get_build_requires_wheel
2025-03-02T08:54:48.7740100Z     return backend.get_requires_for_build_wheel()
2025-03-02T08:54:48.7740360Z            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
2025-03-02T08:54:48.7741870Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_internal/utils/misc.py", line 701, in get_requires_for_build_wheel
2025-03-02T08:54:48.7742480Z     return super().get_requires_for_build_wheel(config_settings=cs)
2025-03-02T08:54:48.7742800Z            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7743410Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_impl.py", line 166, in get_requires_for_build_wheel
2025-03-02T08:54:48.7744010Z     return self._call_hook('get_requires_for_build_wheel', {
2025-03-02T08:54:48.7744300Z            ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7744570Z         'config_settings': config_settings
2025-03-02T08:54:48.7744800Z         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7745020Z     })
2025-03-02T08:54:48.7745160Z     ^^
2025-03-02T08:54:48.7745630Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_impl.py", line 321, in _call_hook
2025-03-02T08:54:48.7746180Z     raise BackendUnavailable(data.get('traceback', ''))
2025-03-02T08:54:48.7746590Z pip._vendor.pyproject_hooks._impl.BackendUnavailable: Traceback (most recent call last):
2025-03-02T08:54:48.7748950Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 77, in _build_backend
2025-03-02T08:54:48.7749600Z     obj = import_module(mod_path)
2025-03-02T08:54:48.7750080Z   File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/importlib/__init__.py", line 88, in import_module
2025-03-02T08:54:48.7750630Z     return _bootstrap._gcd_import(name[level:], package, level)
2025-03-02T08:54:48.7750970Z            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-03-02T08:54:48.7751300Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2025-03-02T08:54:48.7751700Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2025-03-02T08:54:48.7752090Z   File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
2025-03-02T08:54:48.7752440Z ModuleNotFoundError: No module named 'backend'
2025-03-02T08:54:48.7752610Z 
2025-03-02T08:54:49.2236550Z ##[error]Process completed with exit code 2.
2025-03-02T08:54:49.2306180Z Cleaning up orphan processes

```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -6,3 +6,4 @@ include VERSION_NUMBER
 include CMakeLists.txt
 include requirements.txt
 include requirements-reference.txt
+include backend.py

